dds/DCPS/DataReaderImpl_T.h

```diff
@@ -1674,7 +1674,7 @@ void store_instance_data(
       return;
     }
 
-    // first find the instance mapin the participant instance map.
+    // first find the instance map in the participant instance map.
```

Comment:
Let's remove this comment block.  It's a combination of saying exactly what the code already says and then an incomplete thought at the end.